frontend/src/api/index.ts

```diff
@@ -130,7 +130,7 @@ class RequestHttp {
                                 response.status,
                                 response.data && response.data['message'] ? response.data['message'] : '',
                             );
-                            return;
+                            return Promise.reject(error);
                         default:
                             globalStore.isLogin = false;
                             globalStore.errStatus = 'code-' + response.status;
```

Comment:
The code appears to be a part of an HTTP request handler that processes different response types and returns appropriate results or errors. Here are some observations and possible improvements:

1. **Return Statement**: The original line `return;` is incorrect because it doesn't handle any specific error conditions when the status code doesn't match expected values like `2xx`.

   ```javascript
-                return;
   +               // Handle other unexpected statuses with proper error handling
+            }
   ```

2. **Error Handling for Non-Expected Statuses**: The current logic only returns for unexpected statuses (other than `2xx`). You might want to add a more general error handling mechanism to catch all other non-expected cases.

3. **Consistency Across Different Responses**: It's generally better to follow a consistent pattern across different parts of the function. For example:

   ```javascript
             return Promise.reject(new Error(`Server responded with an unexpected status: ${response.status}`));
         default:
             globalStore.isLogin = false;
             globalStore.errStatus = 'code-' + response.status;
+        return Promise.reject(new Error('Failed to process GET/PUT requests'));
     };

     this.handlePostResponse = async ({ response }) => {
-        console.log(response.statusText, response.text(), response.status);
+        if ([200, 201].includes(response.status)) { // Add common success codes here
+            const data = await response.json();
+            Logger.info(data); // Log or use the data accordingly
+            return data;
+        } else {
+            return Promise.reject(new Error(`POST request failed, received status: ${response.status}`)); // Specific to POST requests
```

4. **Promise Rejections**: Ensure that every branch of your asynchronous functions correctly rejects promises. This provides a clear indication of what went wrong.

These changes aim to enhance the robustness and maintainability of the code by handling various scenarios more gracefully.